src/renderers/dialog-renderer.js

```diff
@@ -1,7 +1,9 @@
 import {dialogOptions} from '../dialog-options';
+import {DOM} from 'aurelia-pal';
+import {Container, transient} from 'aurelia-dependency-injection';
 
-let currentZIndex = 1000;
-
+let containerTagName = 'ai-dialog-container';
+let overlayTagName = 'ai-dialog-overlay';
 let transitionEvent = (function() {
   let t;
   let el = document.createElement('fakeelement');
```

Comment:
Yes. @jedd-ahyoung Please remove any document or window references and use the PAL instead.